src/auth/index.ts

```diff
@@ -445,7 +448,7 @@ class Auth {
    * @param {String} parameters.authenticatorId The ID of the authenticator to challenge.
    * @returns {Promise}
    */
-  multifactorChallenge(parameters = {}) {
+  multifactorChallenge(parameters: MultiFactorChallengeOptions) {
```

Comment:
Is it expected for the casing to not be consistent ? (multifactor vs Multi**F**actor)